enciv-transcribe.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Danger! you should not check in private_key and things into github. I see you added it to .gitignore but that was probably after there was a git add .

You will need to move the file out of the directory, git another git add ., and then a git commit -m "removed enciv-transcribe.json" in order to get rid of it.

I'm not sure how this code getting the keys into the api call - are they set in env variables somewhere? We'll need to document it somehow so people can set it up. (like me). 

Also - you are going to need to get new keys - these are public.

Reply:
I was thinking of adding installing instructions into the  readme after we do the merge to master. To get a key you need to put your credit card info, i hope this isnt a barrier for new developers.